tests/playwright/sanity/core/app/modules/onboarding/onboarding.data.json

```diff
@@ -0,0 +1,11 @@
+{
+    "createAccountButtonText" : "Create my account",
+    "disclaimerNotice" : "* You can switch your theme later on",
+    "importElementorPopupTitle" : "Elementor | Upload and Install Elementor Pro",
+    "signIntoElementorPopupTitle" : "Sign in – My Account",
+    "upgradeFeaturesList" : [
+        "90+ Basic & Pro widgets",
+        "300+ Basic & Pro templates",
+        "Premium Support"
+    ]
+}
```

Comment:
Please add new line at the end of the file.

Reply:
The extra line hasn't been added yet. 

Can you please not resolve conversations? I prefer to resolve the conversations which I started as a reviewer.